### PR TITLE
chembl_webresource_client

### PR DESCRIPTION
**Pull request by simonbray**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/chembl_webresource_client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - easydict
+    - grequests
+    - pip
+    - python >=3.5,<3.7
```
Comment (marcelotrevisani) [inline recipes/chembl_webresource_client/meta.yaml]: you can pin just ``3.6`` then
Why this project does not support Python 3.7?

you can do something like:
```suggestion
    - python 3.6.*
```

File: recipes/chembl_webresource_client/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - requests_cache
+  run:
+    - easydict
+    - grequests
+    - python >=3.5,<3.7
```
Comment (marcelotrevisani) [inline recipes/chembl_webresource_client/meta.yaml]: same here
```suggestion
    - python 3.6.*
```

File: recipes/chembl_webresource_client/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/5d/43/bf0a4d475be9b16f1cbecf0dfe5cb2652305ad239d726a9238d1c4d1713d/chembl-webresource-client-0.9.31.tar.gz
+  sha256: 7a51a158271f569c1b20354541ee10988f060bcdbe1ce9e76f0a0d76e57beb10
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (marcelotrevisani) [inline recipes/chembl_webresource_client/meta.yaml]: I think this recipe can be ``noarch``, I think if you add the ``entry_points`` that should be fine
```suggestion
  noarch: python
  entry_points:    
    - chembl_ids=chembl_webresource_client.scripts.chembl_ids:main
    - chembl_sim=chembl_webresource_client.scripts.chembl_sim:main
    - chembl_sub=chembl_webresource_client.scripts.chembl_sub:main
    - chembl_m2t=chembl_webresource_client.scripts.chembl_m2t:main
    - chembl_t2m=chembl_webresource_client.scripts.chembl_t2m:main
    - chembl_act=chembl_webresource_client.scripts.chembl_act:main
```

File: recipes/chembl_webresource_client/meta.yaml (hunk 1 above)
Comment (simonbray) [reply recipes/chembl_webresource_client/meta.yaml]: Thanks for the quick review.
There seem to be a couple of reasons, see https://github.com/chembl/chembl_webresource_client/issues/40

File: recipes/chembl_webresource_client/meta.yaml (hunk 3 above)
Comment (simonbray) [reply recipes/chembl_webresource_client/meta.yaml]: Sure, let's try

File: recipes/chembl_webresource_client/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    - easydict
+    - grequests
+    - pip
+    - python 3.6.*
+    - requests_cache
+  run:
+    - easydict
+    - grequests
+    - python 3.6.*
+    - requests_cache
```
Comment (marcelotrevisani) [inline recipes/chembl_webresource_client/meta.yaml]: Please add ``six``
```suggestion
    - requests_cache
    - six
```

File: recipes/chembl_webresource_client/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
+{% set name = "chembl_webresource_client" %}
+{% set version = "0.9.31" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/5d/43/bf0a4d475be9b16f1cbecf0dfe5cb2652305ad239d726a9238d1c4d1713d/chembl-webresource-client-0.9.31.tar.gz
```
Comment (marcelotrevisani) [inline recipes/chembl_webresource_client/meta.yaml]: One small improvement here. Could you please change it to:
```suggestion
  url: https://files.pythonhosted.org/packages/5d/43/bf0a4d475be9b16f1cbecf0dfe5cb2652305ad239d726a9238d1c4d1713d/chembl-webresource-client-{{ version }}.tar.gz
```

**Comment by marcelotrevisani** [comment]
There is a bug on this project for windows platform. Could you please open a ticket about it?
For now, you can just skip the windows, but please let the ``entry_points``

File: recipes/chembl_webresource_client/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+  sha256: 7a51a158271f569c1b20354541ee10988f060bcdbe1ce9e76f0a0d76e57beb10
+
+build:
+  number: 0
+  noarch: python
```
Comment (marcelotrevisani) [inline recipes/chembl_webresource_client/meta.yaml]:
```suggestion
  skip: true  # [win]
```

**Pull request merged**